ODT/ODTUtility.php

```diff
@@ -198,7 +198,12 @@ public static function getImageSize($src, $maxwidth=NULL, $maxheight=NULL){
             $info = getimagesize($src);
         } else {
             // FIXME: Add cache support for downloaded images.
-            $fetch = (new DokuHTTPClient())->get($src);
+            if (class_exists('dokuwiki\HTTP\DokuHTTPClient')) {
+                $http = new dokuwiki\HTTP\DokuHTTPClient();
+            } else {
+                $http = new DokuHTTPClient();
+            }
+            $fetch = @$http->get($src);
```

Comment:
why is suppression of errors needed? I not fan of it, in other project they can avoid it. So I'm surprised to see used here...

Reply:
This is needed because if URL is unreachable (e.g. there's no connectivity to the external server because it's offline or it's a broken link, or proxy settings isn't set). There's a verification of ``$fetch`` content to check if it's empty response after this check so the ``@`` suppresses the error display when the PHP directive ``display_errors`` is set to **On**.